department-of-veterans-affairs/caseflow
# Colocated Queue | Place hold

Connects #6082, #6083, #6084

Dropdown action: #6082
Place hold screen: #6083
Confirmation: #6084

### Description
Adds flow for colocated user to place task on hold

### Acceptance Criteria 
- [ ] Place hold flow sets `status: on_hold` and `on_hold_duration: duration`

### Testing Plan
1. Go to `/queue` as `BVALSPORER`, go to case details for case in New tab (default tab)
2. Select 'Place hold' from actions dropdown
3. Choose hold / enter custom hold, click "Place case on hold"
4. Confirm case moves to "On hold" tab, "Days on Hold" column reflects hold duration

<details><summary>gif</summary>
<img src="https://user-images.githubusercontent.com/8533004/44350637-e2e74d00-a46d-11e8-870d-bde32bc4a42a.gif">
</details>
